modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,88 @@
+
+== Configuration
+
+=== First steps
+
+Log in as user `admin`. The default password is `admin` until you change it. In the upper right-hand corner, click on menu:Admin[Settings]. In the left hand navigation bar you find personal configuration options and admin-specific options. Under menu:Personal[General] you can change the password of you admin user.
+
+For more information on the Web interface, refer to the xref:user_manual:webinterface.adoc[Web UI] documentation.
+
+=== Configure your email.
+
+Email is an essential tool in ownCloud for notifying users, user registration, password resets, etc. Logged-in as admin, you can configure email in the menu:Admin[General] settings.
+
+For more information, refer to the section xref:configuration/server/email_configuration.adoc[Email Configuration].
+
+=== Cron
+
+On the same page (menu:Admin[General]) you find the Cron settings.
+
+Make sure Cron remains enabled. This daemon takes care of service jobs in 15-minute intervals.
+
+NOTE: In general, we recommend to keep the default settings in the admin section.
+
+== User Management
+
+== Enabling ownCloud Apps
+
+With the free community edition, you can use all apps that are published under the GPL or similar public licenses, e.g. the GNU Affero General Public License. These you can simply install and enjoy.
+In the upper left corner, click on menu:Files[Market]. Here you can have your pick.
+
+A bundle of Enterprise Apps is published under the ownCloud Commercial License and only available with the Enterprise Edition. To see what's not included in the free community edition, select `App Bundles` in the left-hand navigation bar.
+
+== Enterprise Edition and Licensing
+
+If you need more users than five or want to use enterprise features, obtain an Enterprise License and possibly additonal user packages via the QNAP Store and activate them in the QNAP License Manager.
+
+Licenses are valid for a year. If you don't renew them, only the first five users created will remain enabled as well as non-enterprise apps. Should you decide to buy an Enterprise license and user packages again, you can enable the users you want to become active again in the Web interface
+
+== Backing up the Database
+
+== Console Management
+
+== occ commands
+
+== Trouble Shooting
+
+Via the QuLog Center utility you can check the log entries.
+
+The event notifications in the top tool bar will also tell you if something has gone wrong.
+
+If you forgot to install the Container Station, an error message will pop up during the installation of ownCloud. Click on the link "System Event Log" in the pop-up window to find out what actually went wrong or hit btn:[OK] and install the Container Station. Then start the installation of ownCloud again. 
+
+== FAQ
+
+. How to resize a storage pool or add a new disk
+
+. How to grant access to guest users
```

Comment:
prerequisites:
- email server is configured and working
- guest app is installed from marketplace / activated in the apps settings (since it is installed as default)
- guest user must be able to access the ownCloud (Same network, exposed to the internet.... this will need some more description in an own chapter / FAQ )

grant access:
- share a file / folder with the email address of guest user